README.md

```diff
@@ -8,14 +8,157 @@ Based on steverhoades/oauth2-openid-connect-client.
 
 ## Usage
 
-Publish config with artisan command `vender:publish` for provider `LaravelOIDCAuth\OIDCAuthServiceProvider`, and fill it.
+Publish config with artisan command `vender:publish` for provider `LaravelOIDCAuth\OIDCAuthServiceProvider`, and fill
+it.
 
-Use `LaravelOIDCAuth\Authenticate` as auth middleware to redirect directly to OIDC login automatically. Alternatively, you can get login url by yourself and store state string into session storage `oidc-auth.state`, see that middleware.
+Use `LaravelOIDCAuth\Middleware\Authenticate` as auth middleware to redirect directly to OIDC login automatically. Alternatively,
+you can get login url by yourself and store state string into session storage `oidc-auth.state`, see that middleware.
 
 ### `config/auth.php`
 
 Set guard driver to `oidc-auth-session` to remove tokens from session storage on logout.
 
-Set provider driver to `oidc-auth-session` to make `Auth::user()` return a `LaravelOIDCAuth\OIDCUser` authenticable from saved token.
+Set provider driver to `none`, user provider is not suitable.
```

Comment:
> is not suitable

is not needed

> Set provider driver to `none`

Removing that key may be better

Reply:
I agree.